flowsa/common.py

```diff
@@ -84,16 +84,27 @@ def load_crosswalk(crosswalk_name):
     return cw
 
 
-def load_yaml_dict(filename):
+def load_yaml_dict(filename, flowbytype=None):
     """
     Load the information in 'source_catalog.yaml'
     :return: dictionary containing all information in source_catalog.yaml
     """
-    folder = datapath if filename == 'source_catalog' else sourceconfigpath
+    if filename == 'source_config':
```

Comment:
@matthewlchambers Line 92 sound read `if filename == 'source_catalog:`

Reply:
Fixed! Glad you caught that!